### ISSUE TITLE
django integration

### ISSUE THREAD

**Issue by devxpy** [codelv/enaml-web#8]
Is there any way to integrate this with django currently?

**Comment by frmdstryr** [comment]
Yes.  You can create a `LxmlApplication` which has no server component and then render pages for any application (django, flask, etc..) or save them to a file to make a static site.

I pushed an example here https://github.com/codelv/enaml-web/tree/master/examples/static_site

**Comment by frmdstryr** [comment]
Some of them are here https://github.com/codelv/enaml-web/blob/master/examples/simple_site/views/blocks.enaml

It would be really nice to have separate libraries with all of the components for the popular css frameworks ([materialize](https://semantic-ui.com/),  [bootstrap](https://getbootstrap.com/), [semantic-ui](https://semantic-ui.com/) etc..).

**Comment by devxpy** [comment]
It would be even nicer if we could write a code generator for this :D

**Comment by devxpy** [comment]
Are you interested in uploading to pypi?

Please check out my pull request that'll make it easier for you to do it

https://github.com/codelv/enaml-web/pull/9
